CMakeLists.txt

```diff
@@ -109,8 +109,17 @@ if (BUILD_SDF)
   ign_find_package(ignition-utils1 VERSION REQUIRED)
   set(IGN_UTILS_VER ${ignition-utils1_VERSION_MAJOR})
 
+  ########################################
+  # Find ignition common
+  ign_find_package(ignition-common4 COMPONENTS graphics REQUIRED_BY usd)
+  set(IGN_COMMON_VER ${ignition-common4_VERSION_MAJOR})
+
+  ########################################
+  # Find PXR
+  ign_find_package(pxr REQUIRED_BY usd PKGCONFIG pxr)
```

Comment:
I'm not sure if the `PKGCONFIG pxr` portion should be here. If I didn't include this, I would see the following warning: https://github.com/ignitionrobotics/ign-cmake/blob/a1eb36a9a6283bb1fe5b4c755a2b3669572e0c08/cmake/IgnUtils.cmake#L378-L391

Can someone confirm if adding `PKGCONFIG pxr` is correct or not?